src/StockportWebapp/Views/Shared/Profile.cshtml

```diff
@@ -18,74 +20,7 @@
-                        </div>
-                    </div>
-                }
-                <div class="grid-@width tablet-grid-@width mobile-grid-100">
-                    <div class="mobile-push-10 headings">
-                        <h2>@Model.Profile.Title</h2>
-                        <span class="summary">@Model.Profile.Subtitle</span>
-                    </div>
-                    <div class="grid-100 grid-parent body">
-                        <div class="grid-10 tablet-grid-10 mobile-grid-10"><span class="green open fa fa-2x fa-quote-left" aria-hidden="true"></span></div>
-                        <div class="grid-80 tablet-grid-80 mobile-grid-80 main-body">
-                            <p>@Model.Profile.Teaser</p>
-                        </div>
-                        <div class="grid-10 tablet-grid-10 mobile-grid-10"></div>
-                    </div>
-                    <div class="grid-100 grid-parent closing-quote">
-                        <div class="grid-10 tablet-grid-10 mobile-grid-10 tablet-push-90 mobile-push-90 push-90"><span class="green fa fa-2x fa-quote-right" aria-hidden="true"></span></div>
-                    </div>
-                    <p class="read-more">Read more</p>
-                </div>
-            </a>
-        </div>
-    }
-    else {
-        <div class="profile profile-success-story-new">
-            @{ 
-                var width = "100";
-            }
-            @if (!string.IsNullOrEmpty(Model.Profile.Image.Url))
-            {
-                width = "75";
-                <div class="grid-25 tablet-grid-25 mobile-grid-100">
-                    <div class="profile-image-mask">
-                        <img src="@Model.Profile.Image.Url" alt="@Model.Profile.Title" />
-                    </div>
-                </div>
-            }
-            <div class="grid-@width tablet-grid-@width mobile-grid-100">
-                <div class="mobile-push-10 headings">
-                    <h2>@Model.Profile.Title</h2>
-                    <span class="summary">@Model.Profile.Subtitle</span>
-                </div>
-                <div class="grid-100 grid-parent body">
-                    <div class="grid-10 tablet-grid-10 mobile-grid-10"><span class="green open fa fa-2x fa-quote-left" aria-hidden="true"></span></div>
-                    <div class="grid-80 tablet-grid-80 mobile-grid-80 main-body">
-                        <p>@Model.Profile.Teaser</p>
-                    </div>
-                    <div class="grid-10 tablet-grid-10 mobile-grid-10"></div>
-                </div>
-                <div class="grid-100 grid-parent closing-quote">
-                    <div class="grid-10 tablet-grid-10 mobile-grid-10 tablet-push-90 mobile-push-90 push-90"><span class="green fa fa-2x fa-quote-right" aria-hidden="true"></span></div>
-                </div>
-            </div>
-        </div>
-    }
-</feature>
\ No newline at end of file
+}
+else {
```

Comment:
Slightly pedantic but pop the brace on the next line (or move the if opening bracket up a line)